dart/lib/src/protocol/sdk_version.dart

```diff
@@ -36,11 +36,11 @@ import 'sentry_package.dart';
 class SdkVersion {
   /// Creates an [SdkVersion] object which represents the SDK that created an [Event].
   SdkVersion({
-    @required this.name,
-    @required this.version,
-    List<String> integrations,
-    List<SentryPackage> packages,
-  })  : assert(name != null || version != null),
+    required this.name,
```

Comment:
`SdkVersion` required name and version which make it unlikely that anyone just passed on of them like the assert suggests.
Currently they are both non-nullable.

Reply:
name and version are required, non-nullable https://develop.sentry.dev/sdk/event-payloads/sdk/
the thing is, we set this automatically via the SDK, probably no one is overwriting this.
`integrations` and `packages`, nullable